### PR TITLE
[ASCellNode] focusStyle mapping

### PR DESCRIPTION
**Pull request by alexhillc**
Adds support for setting `focusStyle` on an `ASCellNode`.

### CODE REVIEW

**Comment by alexhillc** [comment]
Hey @maicki thanks for reviewing. I've been building against the tvOS SDK, so I must have missed it. Fixed in the latest commit!

File: Source/ASTableView.mm [hunk 1]
```diff
@@ -114,12 +114,15 @@ - (void)setElement:(ASCollectionElement *)element
   
   if (node) {
     self.backgroundColor = node.backgroundColor;
-    self.selectionStyle = node.selectionStyle;
     self.selectedBackgroundView = node.selectedBackgroundView;
     self.separatorInset = node.separatorInset;
     self.selectionStyle = node.selectionStyle;
     self.accessoryType = node.accessoryType;
     
+    if (AS_AVAILABLE_IOS(9.0)) {
```
Comment (maicki) [inline Source/ASTableView.mm]: To be consistent can you change that to 9

**Comment by nguyenhuy** [comment]
@alexhillc We've dropped iOS 8 support (#743). Could you please update this diff one more time? Thank you!

**Comment by alexhillc** [comment]
@nguyenhuy, done!

**Comment by maicki** [comment]
Awesome thanks all. Let's get that PR in than.

**Pull request merged**